packages/shared_preferences/shared_preferences/example/windows/flutter/generated_plugin_registrant.cc

```diff
@@ -4,4 +4,6 @@
 
 #include "generated_plugin_registrant.h"
 
-void RegisterPlugins(flutter::PluginRegistry* registry) {}
+
+void RegisterPlugins(flutter::PluginRegistry* registry) {
+}
```

Comment:
Needs reverting.

Reply:
The act of running `flutter pub get` is what seems to trigger this change, rather than any formatting that I'm imposing. Reverted this, and hopefully it sticks!

Reply:
Yes, it's autogenerated by `flutter`. The format that is checked in, and that the CI enforces on the codebase in general, is the clang-format output (for our particular configuration).

The best solution would be for the flutter/plugins format enforcement to have an ignore-list for generated files, because the tug of war between clang-format's output and the generated file's format is slightly annoying (and pointless).

Reply:
The easy way to fix this in the short term is to `git checkout master -- <paths to the generated files>` then `git commit`